ocp-config/prov-app/cm.yml

```diff
@@ -138,53 +139,78 @@ objects:
-      jenkinspipeline.quickstarter.dsRshiny.desc=Data Science - R-Shiny
-      jenkinspipeline.quickstarter.dsRshiny.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/ds-rshiny/Jenkinsfile
-      jenkinspipeline.quickstarter.dsMlService.desc=Data Science - Machine Learning
-      jenkinspipeline.quickstarter.dsMlService.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/ds-ml-service/Jenkinsfile
+      jenkinspipeline.quickstarter.ds-jupyter-notebook.desc=Data Science - Jupyter Notebook
+      jenkinspipeline.quickstarter.ds-jupyter-notebook.repo=ods-quickstarters
+
+      jenkinspipeline.quickstarter.ds-rshiny.desc=Data Science - R-Shiny
+      jenkinspipeline.quickstarter.ds-rshiny.repo=ods-quickstarters
+
+      jenkinspipeline.quickstarter.ds-ml-service.desc=Data Science - Machine Learning
+      jenkinspipeline.quickstarter.ds-ml-service.repo=ods-quickstarters
 
       # Frontend quickstarters
-      jenkinspipeline.quickstarter.feAngular.desc=Frontend - Angular
-      jenkinspipeline.quickstarter.feAngular.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/fe-angular/Jenkinsfile
-      jenkinspipeline.quickstarter.feReact.desc=Frontend - React
-      jenkinspipeline.quickstarter.feReact.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/fe-react/Jenkinsfile
-      jenkinspipeline.quickstarter.feVue.desc=Frontend - Vue
-      jenkinspipeline.quickstarter.feVue.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/fe-vue/Jenkinsfile
-      jenkinspipeline.quickstarter.feIonic.desc=Mobile - Ionic
-      jenkinspipeline.quickstarter.feIonic.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/fe-ionic/Jenkinsfile
+      jenkinspipeline.quickstarter.fe-angular.desc=Frontend - Angular
+      jenkinspipeline.quickstarter.fe-angular.repo=ods-quickstarters
+
+      jenkinspipeline.quickstarter.fe-react.desc=Frontend - React
+      jenkinspipeline.quickstarter.fe-react.repo=ods-quickstarters
+
+      jenkinspipeline.quickstarter.fe-vue.desc=Frontend - Vue
+      jenkinspipeline.quickstarter.fe-vue.repo=ods-quickstarters
+
+      jenkinspipeline.quickstarter.fe-ionic.desc=Mobile - Ionic
+      jenkinspipeline.quickstarter.fe-ionic.repo=ods-quickstarters
 
       #Other quickstarters
-      jenkinspipeline.quickstarter.beAirflow.desc=Other - Airflow
-      jenkinspipeline.quickstarter.beAirflow.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/airflow-cluster/Jenkinsfile
-      jenkinspipeline.quickstarter.e2eCypress.desc=Other - Cypress E2E test
-      jenkinspipeline.quickstarter.e2eCypress.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/e2e-cypress/Jenkinsfile
-      jenkinspipeline.quickstarter.releaseManager.desc=Other - Releasemanager
-      jenkinspipeline.quickstarter.releaseManager.url=opendevstack/ods-quickstarters.git#${ODS_GIT_REF}/release-manager/Jenkinsfile
+      jenkinspipeline.quickstarter.docker-plain.desc=Docker/Plain
```

Comment:
Please change description to Other - Docker/Plain, because Quickstarters are sorted by description